docs-private/Performance-Testing-Guide.md

```diff
@@ -0,0 +1,203 @@
+   {
+    "id": "operation.amount",
+    "type": "AMOUNT",
+    "text": "Amount",
+    "params": {
+      "amount": "amount",
+      "currency": "currency"
+    }
+   },
+   {
+    "id": "operation.account",
+    "type": "KEY_VALUE",
+    "text": "To Account",
+    "params": {
+      "value": "iban"
+    }
+   }
+   ]', null);
+   ```
+
+   MSSQL
+
+   ```tsql
+   INSERT INTO es_operation_template (id, placeholder, language, title, message, attributes, ui)
+   VALUES (1, 'login', 'en', 'Login Approval', 'Are you logging in to the internet banking?', NULL, NULL);
+
+   INSERT INTO es_operation_template (id, placeholder, language, title, message, attributes, ui)
+   VALUES (2, 'authorize_payment', 'en', 'Payment Approval', 'Please confirm the payment', '[
+   {
+    "id": "operation.amount",
+    "type": "AMOUNT",
+    "text": "Amount",
+    "params": {
+      "amount": "amount",
+      "currency": "currency"
+    }
+   },
+   {
+    "id": "operation.account",
+    "type": "KEY_VALUE",
+    "text": "To Account",
+    "params": {
+      "value": "iban"
+    }
+   }
+   ]', NULL);
+   ```
+
+   Oracle
+
+   ```oracle
+    INSERT INTO es_operation_template (id, placeholder, language, title, message, attributes, ui)
+    VALUES (1, 'login', 'en', 'Login Approval', 'Are you logging in to the internet banking?', NULL, NULL);
+
+    INSERT INTO es_operation_template (id, placeholder, language, title, message, attributes, ui)
+    VALUES (2, 'authorize_payment', 'en', 'Payment Approval', 'Please confirm the payment', '[{"id": "operation.amount", "type": "AMOUNT", "text": "Amount", "params": {"amount": "amount", "currency": "currency"}}, {"id": "operation.account", "type": "KEY_VALUE", "text": "To Account", "params": {"value": "iban"}}]', NULL);
+   ```
+
+## Running the tests
+
+To run the tests we use maven gatling plugin. The methodology of the tests is as follows. There are two simulations:
+
+- Data Preparation
+- Performance Test
+
+There is a config file [.perf_test_config](./../powerauth-load-tests/src/test/resources/.perf_test_config) which needs
+to be sourced before running the tests. This file serves as a configuration file for the testing. More information about
+possible parameters to set is inside the file.
+
+Before running the Performance Test Simulation, which is essentially a load test, **the Data Preparation needs to be run
+at least once to populate the db with some data**. It also generates a JSON file
+to `powerauth-load-tests/src/test/resources` which holds information that is used during the Performance Test.
+
+One may use prepared scripts to easily trigger the tests:
+
+- [run_perf_tests.sh](./../powerauth-load-tests/src/test/resources/run_perf_tests.sh)
+- [run_data_prepare.sh](./../powerauth-load-tests/src/test/resources/run_data_prepare.sh)
+
+The reports are by default generated to `/results/`.
+
+## Useful
+
+- Link to Gatling docs - https://docs.gatling.io
+
+- Generating reports from failed/unfinished tests
+  ```Bash
```

Comment:
🤔 It seems that it is working, but I am surprised by the capital letter, see https://github.com/github-linguist/linguist/blob/master/lib/linguist/languages.yml